src/main/java/io/anserini/search/SimpleGeoSearcher.java

```diff
@@ -24,11 +25,20 @@
   private IndexSearcher searcher = null;
 
   public Result[] searchGeo(Query query, int k) throws IOException {
+    return searchGeo(query, k, null);
```

Comment:
Don't recall from previous PR. Did we want just `search` here?

Reply:
@lintool Naming convention-wise, the previous PR also had the method named `searchGeo.` I chose to call it `searchGeo` instead of `search` to make it analogous to `searchTweets` in `SimpleTweetSearcher,` which also extends `SimpleSearcher.`

Functionally, the previous PR didn't have an overloaded method to support the sort argument and had all the logic in `searchGeo(Query query, int k).` In this PR, I refactored the logic into `searchGeo(Query query, int k, Sort sort)` so `searchGeo(Query query, int k)` becomes a wrapper function for when we don't care about the order of results.

Does this address what you had in mind?

Reply:
Sounds good!